.github/workflows/ci.yml

```diff
@@ -0,0 +1,443 @@
+name: Build and test gem
```

Comment:
Also with the question(s) of how checks show up in the PR UI, this might be better as `name: ci`